includes/Classifai/Blocks/recommended-content-block/register.php

```diff
@@ -46,3 +87,40 @@ function render_block_callback( $attributes, $content, $block ) {
 	include __DIR__ . '/markup.php';
 	return ob_get_clean();
 }
+
+/**
+ * Reset category and taxonomy if usePostTerms is true
+ *
+ * @param array $attributes Block attributes
+ * @return array
+ */
+function use_default_post_terms( $attributes ) {
+	if ( ! empty( $attributes['usePostTerms'] ) && true === $attributes['usePostTerms'] ) {
```

Comment:
I think we can invert the conditions here and return `$attributes` early if the conditions don't match, rather than wrap all this code in a conditional